### PR TITLE
Node addon

### PR DESCRIPTION
**Pull request by mapsam**
Converting this into a node-cpp-skel project! This doesn't incorporate any of the logic in the vtquery/ directory - just gets the bones put together.

cc @springmeyer @GretaCB

### CODE REVIEW

**Comment by springmeyer** [comment]
Looks like this is missing the `.travis.yml`?

**Comment by mapsam** [comment]
These tests won't pass on travis right now since we're trying to include protozero/vtzero/spatial-algorithms from local checkouts. Going to forge ahead since we're 👍 locally and working on master.

**Comment by springmeyer** [comment]
> Going to forge ahead since we're 👍 locally and working on master.

right, 👍

File: vtquery/main.cpp [hunk 1]
```diff
@@ -32,7 +32,7 @@ int main() {
   std::vector<mapbox::geometry::point<std::int64_t>> results;
 
   // read buffer from filesystem
-  std::string filename("../mvt-fixtures/fixtures/017/tile.mvt");
+  std::string filename("/Users/mapsam/Downloads/3044.vector.pbf");
```
Comment (GretaCB) [inline vtquery/main.cpp:35]: @mapsam Just fyi

Comment (mapsam) [reply vtquery/main.cpp:35]: oops! Thanks for catching this

**Pull request merged**